azure/7.0/ha-port1-mgmt/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+// Change to your own azure environment
+subscription_id = "<fill your subscription id>"
```

Comment:
Probably should be renamed tfvars.example, with the tfvars file in .gitignore

Reply:
I don't know why we would put this in gitignore. But if you leave them blank they should work as inputs if that is what you want. Probably just depends on the end-user.